connector/src/rest/connector/libs/tmos/implementation.py

```diff
@@ -0,0 +1,444 @@
+# Global imports
+import logging
+
+# Genie, PyATS, ROBOT imports
+from pyats.connections import BaseConnection
+from rest.connector.utils import get_username_password
+
+# F5 imports
+from icontrol.session import iControlRESTSession
+from icontrol.exceptions import iControlUnexpectedHTTPError
+
+# create a logger for this module
+log = logging.getLogger(__name__)
+
+
+class Implementation(BaseConnection):
```

Comment:
This should inherits from Implementation 

```
   from rest.connector.implementation import Implementation
   class Implementation(Implementation):
```
Then you dont need the __init__ as the generic implementation does this by default.

https://github.com/CiscoTestAutomation/rest/blob/master/connector/src/rest/connector/implementation.py#L47

Reply:
will do.